### PR TITLE
Ported to Python 3

### PR DESCRIPTION
**Pull request by Fethbita**
The whitespace changes are kind of annoying, sorry for that.

### CODE REVIEW

File: source/data/search_serial_in_large_CRL.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def search_certificate(crl_der, serial, dictionary):
 
 def main():
     """Main program"""
-    crl_filename = "www.sk.ee-crl.crl"
+    crl_filename = "source/data/www.sk.ee-crl.crl"
```
Comment (getreu) [inline source/data/search_serial_in_large_CRL.py]: Better revert this change that the script finds its data.

File: source/data/search_serial_in_large_CRL.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def main():
     print()
     print()
 
-    crl_filename = "www.sk.ee-esteid2011.crl"
+    crl_filename = "source/data/www.sk.ee-esteid2011.crl"
```
Comment (getreu) [inline source/data/search_serial_in_large_CRL.py]: Better revert this change that the script finds its data.

File: source/data/search_serial_in_large_CRL-output.txt [hunk 3]
```diff
@@ -8,20 +8,21 @@ crl_signed_content:    (4, 8, 511) 508 Bytes
...
-serial:  1084184741   position: (414, 416, 450)
 serial:  1167825894   position: (192, 194, 228)
+serial:  1167830238   position: (229, 231, 265)
+serial:  1043934336   position: (266, 268, 302)
+serial:  1018259643   position: (303, 305, 339)
 serial:  1018438612   position: (340, 342, 376)
 serial:  1018438937   position: (377, 379, 413)
-serial:  1018259643   position: (303, 305, 339)
-serial:  1167830238   position: (229, 231, 265)
+serial:  1084184741   position: (414, 416, 450)
```
Comment (getreu) [inline source/data/search_serial_in_large_CRL-output.txt:17]: Strange, how come that serials and position has changed?

**Review by getreu — CHANGES_REQUESTED** [review]
Thank you for your work! It looks fine to me.
2 tiny little requests: 2 paths should not change (see comments) and 1 question.

File: source/data/search_serial_in_large_CRL.py (hunk 2 above)
Comment (Fethbita) [reply source/data/search_serial_in_large_CRL.py]: Ah yes, I tried not to include the path changes but I missed them, I'll fix this soon.

File: source/data/search_serial_in_large_CRL-output.txt (hunk 3 above)
Comment (Fethbita) [reply source/data/search_serial_in_large_CRL-output.txt:17]: I think Python3 dictionaries are ordered, so if you add things to it when you go through them you get them in the order you added them in. I remember Python2 being random, I'm not 100% on this though. The serials are still the same, only the orders changed.

Comment (Fethbita) [reply source/data/search_serial_in_large_CRL-output.txt:17]: Oh yes, here's something I found about it: "If you want the same behavior as earlier versions of Python 2.x in both 2.7 (which uses an arbitrary sort order instead) and 3.x (which refuses to sort dicts)," from this [answer](https://stackoverflow.com/a/25675708/6077951)

**Comment by getreu** [comment]
Thank you.
